ui/pages/confirm-transaction-base/gas-details-item/gas-details-item.js

```diff
@@ -88,9 +92,18 @@ const GasDetailItem = ({
         </div>
       }
       subText={t('editGasSubTextFee', [
-        <Box key="editGasSubTextFeeLabel" display="inline-flex">
-          <Box marginRight={1} className="gas-details-item__gasfee-label">
-            <I18nValue messageKey="editGasSubTextFeeLabel" />
+        <Box
+          key="editGasSubTextFeeLabel"
+          display="inline-flex"
+          className={classNames('gas-details-item__gasfee-label', {
+            [`gas-details-item__gas-fee-warning`]: estimateToUse === 'high',
```

Comment:
Was this variable in the past?  I don't think we need these `[]`

Reply:
Good point, that was not needed, I removed it.